dynamiq/storages/vector/weaviate/weaviate.py

```diff
@@ -714,6 +786,16 @@ def delete_documents_by_filters(self, filters: dict[str, Any]) -> None:
         else:
             raise ValueError("No filters provided to delete documents.")
 
+    def delete_documents_by_file_id(self, file_id: str) -> None:
```

Comment:
It seems this one method already exists in Base class, so no need to duplicate it